mist-api/src/routes/genomes/get.js

```diff
@@ -10,6 +10,42 @@ module.exports = function(app, middlewares, routeMiddlewares) {
 	let models = app.get('models'),
 		helper = app.get('lib').RouteHelper.for(models.Genome)
 
+	const taxonomyTextFieldNames = [
+		'version',
+		'superkingdom',
+		'phylum',
+		'class',
+		'order',
+		'family',
+		'genus',
+		'assembly_level',
+	]
+
+	const checkAndGetTerms = (queryValue, target, modelFieldName) => {
```

Comment:
Scrap `modelFieldName` as it is not used in this method. To check that this is defined, please do this in the `processSearch` method body. After removing this parameter, could simply replace this method with a call to  the new one created as specified earlier.

Reply:
OK.